.gitignore

```diff
@@ -179,3 +179,5 @@ data/*
 **lightning_logs/
 outputs/
 wandb/
+dtu_mlops_02476/
```

Comment:
Configuration settings are leaking into the .gitignore. Maybe change the name of the wandb folder.